front_end/src/components/layout/Navbar.js

```diff
@@ -24,7 +25,13 @@ class Navbar extends Component {
     this.props.logoutUser();
   }
 
-  render() {
+  componentWillMount(){
```

Comment:
do we still need this, if we do it through the auth action?

Reply:
yes, cause when we refresh it, it won't call the auth action, we need to call the profile action to reload the `setCurrentUser()` then we can keep the user info. And we need the profile action to get the current profile to pass to `setCurrentUser()`